dartdoc_test/example/README.md

```diff
@@ -0,0 +1,2 @@
+A sample command-line application with an entrypoint in `bin/`, library code
+in `lib/`, and example unit test in `test/`.
```

Comment:
I'm not sure I understand this `README`. Can we have a proper README explaining, how to setup and test things?

Reply:
It is a example app's README, but I made it better :)